### PR TITLE
Implementing Jekyll for BeSLab

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

**Comment by sudhirverma** [comment]
To test this PR

```
1. cd docs
2. bundle install
3. bundle exec jekyll serve
```

File: docs/_docs/01-quick-start-guide.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: "Quick-Start Guide"
+permalink: /docs/quick-start-guide/
+excerpt: "How to quickly install and setup Minimal Mistakes for use with GitHub Pages."
```
Comment (anilsingla) [inline docs/_docs/01-quick-start-guide.md:4]: Need to update excerpt.

File: docs/_docs/01-quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+permalink: /docs/quick-start-guide/
+excerpt: "How to quickly install and setup Minimal Mistakes for use with GitHub Pages."
+last_modified_at: 2024-05-05T08:48:05-04:00
+redirect_from:
+  - /theme-setup/
+toc: true
+---
+
+
+**Note:** BeSLab only suported on Ubuntu based system.
```
Comment (anilsingla) [inline docs/_docs/01-quick-start-guide.md:12]: At present its supporting only Ubuntu. May be we can say it at present only Ubuntu with version 22.04 specifically.

File: docs/_docs/05-configuration.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+toc: true
+---
+
+## Description
+
+This configuration file, named `genesis.yaml`, is utilized by the BLIman (Be-Secure Lab Instantiator Manager) tool to seed a BeSLab instance. BeSLab is a platform designed for setting up security analysis labs efficiently.
+
+## Usage
+
+The `genesis.yaml` file contains various configuration options that define the setup of the BeSLab instance. These options include specifying the lab type, lab name, version of BeSMan (Be-Secure Management), deployment mode, version of BeSLab, URLs and paths for retrieving necessary files, configuration for Java, dashboard tool settings, and more.
```
Comment (anilsingla) [inline docs/_docs/05-configuration.md:15]: Add Code Collaboration Tool as well. Code Collaboration tool is the main component of BeSLab.

File: docs/_pages/about.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+permalink: /about/
+title: "BeS (Be-Secure) Initiative Overview"
+excerpt: "Minimal Mistakes is a flexible two-column Jekyll theme."
```
Comment (anilsingla) [inline docs/_pages/about.md:4]: update excerpt

File: docs/_pages/lorem-ipsum.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Lorem Ipsum"
```
Comment (anilsingla) [inline docs/_pages/lorem-ipsum.md:2]: Do we need this page?

File: docs/_pages/page-a.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (anilsingla) [inline docs/_pages/page-a.md:1]: Do we need this page ? If so do we need to change content ?

File: docs/_pages/page-b.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (anilsingla) [inline docs/_pages/page-b.md:1]: we can delete this page if not needed.

File: docs/_pages/pets.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (anilsingla) [inline docs/_pages/pets.md:1]: I think extra page please check.

File: docs/_pages/portfolio-archive.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (anilsingla) [inline docs/_pages/portfolio-archive.md:1]: Do we need portfolio?

File: docs/_pages/sample-page.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (anilsingla) [inline docs/_pages/sample-page.md:1]: I think this page not needed.

File: docs/_pages/recipes-archive.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (anilsingla) [inline docs/_pages/recipes-archive.md:1]: What is this page for ?

File: docs/_pages/splash-page.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (anilsingla) [inline docs/_pages/splash-page.md:1]: I think we need to update this page to display Be-Secure branding. Check once.

File: docs/_pages/terms.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (anilsingla) [inline docs/_pages/terms.md:1]: We need to define our terms and policy I think. Can we check with Hari or Vinod for it?

File: docs/_posts/2024-05-14-post-exclude-search.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
+---
+title:  "BeS (Be-Secure) Initiative Overview"
+search: false
+categories: 
+  - Jekyll
```
Comment (anilsingla) [inline docs/_posts/2024-05-14-post-exclude-search.md:5]: I think we need to change the category to something else.

**Review by anilsingla — CHANGES_REQUESTED** [review]
Please check the review comments.

**Comment by anilsingla** [comment]
Looks good to me

**Comment by anilsingla** [comment]
Looks Good!!

**Review by anilsingla — APPROVED** [review]

**Pull request merged**